.github/workflows/matomo-tests.yml

```
@@ -54,10 +61,11 @@ jobs:
           submodules: true
           path: matomo
       - name: running tests
-        uses: matomo-org/github-action-tests@main
+        uses: matomo-org/github-action-tests@mariadb
```

```suggestion
        uses: matomo-org/github-action-tests@main
```

This needs to be changed back once https://github.com/matomo-org/github-action-tests/pull/16 has been merged.